proto/dwtools/abase/l0.test/Long.test.s

```diff
@@ -13427,14 +13427,80 @@ function longFromRange( test )
   });
 
 };
+
 //
 
-function longFromRange1( test )
+function longFromRangeWithStep( test )
 {
+  test.case = 'equal numders of range';
+  var got = _.longFromRangeWithStep( [ 1, 1 ], 2 );
+  var expected = [ ];
+  test.identical( got, expected );
 
-  console.log('Hello world');
+  test.case = 'valid range with float step';
+  var got = _.longFromRangeWithStep( [ 0, 10 ], 1.5 );
+  var expected = [ 0, 1.5, 3, 4.5, 6, 7.5, 9 ];
```

Comment:
Please, try to make the expected result with 3-4 elements. It's enough for test case and it is more readable.